src/background.js

```diff
@@ -1,4 +1,51 @@
 import './activity-logger/background'
+import { filterVisitsByQuery } from './search'
+import niceTime from './util/nice-time'
+
+const makeSuggestion = (query, suggest) => {
```

Comment:
Good idea, with that we can mitigate a slow search and people get feedback that something is happening.